components/Table/TableKDA.css

```diff
@@ -0,0 +1,19 @@
+@import '../palette.css';
+@import "./TablePercent.css";
+
+:export {
```

Comment:
no need to export here since in exports in palette.css, I think `low` can be exported as yellowOrange